sources/europe/fr/Brest-Metropole-Orthophoto2024.geojson

```diff
@@ -0,0 +1,442 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "Brest_2023_2024",
+        "name": "Brest Métropole Orthophoto 2023/2024",
+        "type": "wmts",
+        "url": "https://geo.brest-metropole.fr/arcgis/rest/services/public/PhotoAerienne/MapServer/WMTS?",
+        "category": "photo",
+        "start_date": "2023-04-08",
+        "end_date": "2024-04-12",
+        "country_code": "FR",
+        "attribution": {
+            "text": "Brest M\u00e9tropole - 2023-2024",
+            "url": "https://geo.brest-metropole.fr/portal/apps/sites/#/geopaysdebrest/pages/donnees-photographies-aeriennes",
+            "required": true
+        },
+        "max_zoom": 22,
+        "license_url": "https://www.etalab.gouv.fr/licence-ouverte-open-licence",
+        "description": "2023 for roughly the city of Brest, 2024 for the other cities of the area",
+        "best": true,
+        "logo-image": "https://osmlab.github.io/editor-layer-index/sources/europe/fr/Brest_Metropole_logo.png"
```

Comment:
it's just `icon` https://github.com/osmlab/editor-layer-index/blob/gh-pages/CONTRIBUTING.md#icon

```suggestion
        "icon": "https://osmlab.github.io/editor-layer-index/sources/europe/fr/Brest_Metropole_logo.png"
```